### ISSUE TITLE
How to train model using multi-gpus

### ISSUE THREAD

**Issue by Pattaro** [rohban-lab/Knowledge_Distillation_AD#3]
while using DataParall to train modelfind ，I found that the memory of CUDA has been increasing

**Comment by SMSD75** [comment]
Hi, 

We will work on it to provide a multi-GPU version as well. However, if you are in hurry, you can convert the code to a multi-GPU setting yourself. I guess it wouldn't be hard.

**Comment by Pattaro** [comment]
> Hi,
> 
> We will work on it to provide a multi-GPU version as well. However, if you are in hurry, you can convert the code to a multi-GPU setting yourself. I guess it wouldn't be hard.

yeah, I implement it, but the verification found that the AUC of MVTEC was reduced by 5%, so I still hope to get the official version
